### DISCUSSION TITLE
How to override shadow color for a 3rd party library?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#9899] [Help]
So my third party library (PrimeVue) applies this `box-shadow` to an element and I tried to override it with `@apply shadow-primary-500` but nothing happened.

````
.p-checkbox:not(.p-checkbox-disabled) .p-checkbox-box.p-focus {
  @apply shadow-primary-500;
}
````

But this is the result, as can be seen the library does not actually use the `--tw-shadow-color` variable, so I don't know how I can override only the color of the shadow.

**.p-checkbox:not(.p-checkbox-disabled) .p-checkbox-box.p-focus {**
  ~--tw-shadow-color: #86efac !important;~
  ~--tw-shadow: var(--tw-shadow-colored) !important;~
  box-shadow: 0 0 0 2px #ffffff, 0 0 0 4px #ff0000, 0 1px 2px 0 rgb(0 0 0 / 0%) !important;
**}**

**Comment by wongjn** [comment]
Try applying a [box-shadow utility](https://tailwindcss.com/docs/box-shadow) with the box shadow color – applying a box shadow color by itself won't have any effect.
